packages/expo/src/__tests__/Brightness-test.ts

```diff
@@ -0,0 +1,87 @@
+import { NativeModules } from 'react-native';
+
+import * as Brightness from '../Brightness';
+import { mockPlatformAndroid, mockPlatformIOS, unmockAllProperties } from '../../test/mocking';
+
+it(`clamps the brightness value in setBrightnessAsync`, async () => {
+  await Brightness.setBrightnessAsync(5);
+  expect(NativeModules.ExpoBrightness.setBrightnessAsync).toHaveBeenLastCalledWith(1);
+  await Brightness.setBrightnessAsync(-1);
+  expect(NativeModules.ExpoBrightness.setBrightnessAsync).toHaveBeenLastCalledWith(0);
+});
+
+it(`throws when setBrightnessAsync is called with an unsupported type`, async () => {
+  await expect(Brightness.setBrightnessAsync(NaN)).rejects.toThrowError(TypeError);
+  await expect(Brightness.setBrightnessAsync('test' as any)).rejects.toThrowError(TypeError);
+});
+
+it(`clamps the brightness value in setSystemBrightnessAsync`, async () => {
+  mockPlatformAndroid();
+  await Brightness.setSystemBrightnessAsync(5);
+  expect(NativeModules.ExpoBrightness.setSystemBrightnessAsync).toHaveBeenLastCalledWith(1);
+  await Brightness.setSystemBrightnessAsync(-1);
+  expect(NativeModules.ExpoBrightness.setSystemBrightnessAsync).toHaveBeenLastCalledWith(0);
+  unmockAllProperties();
+});
+
+it(`throws when setSystemBrightnessAsync is called with an unsupported type`, async () => {
+  await expect(Brightness.setSystemBrightnessAsync(NaN)).rejects.toThrowError(TypeError);
+  await expect(Brightness.setSystemBrightnessAsync('test' as any)).rejects.toThrowError(TypeError);
+});
+
+it(`does nothing if setSystemBrightnessModeAsync is called with BrightnessMode.UNKNOWN`, async () => {
+  mockPlatformAndroid();
+  await Brightness.setSystemBrightnessModeAsync(Brightness.BrightnessMode.UNKNOWN);
+  expect(NativeModules.ExpoBrightness.setSystemBrightnessModeAsync).not.toHaveBeenCalled();
+  unmockAllProperties();
+});
+
+describe('  system brightness', () => {
+  beforeAll(() => {
+    mockPlatformIOS();
+  });
+
+  afterAll(() => {
+    unmockAllProperties();
+  });
+
+  it('getSystemBrightnessAsync calls getBrightnessAsync', async () => {
```

Comment:
Backtick quotes, also, we should pick whether the test descriptions read as verb phrases ("it calls getBrightnessAsync from getSystemBrightnessAsync" or using a nested `describe('getSystemBrightnessAsync', ...)`) or using `test()` instead of `it()` without any particular part of speech.